### PR TITLE
Added magnifier

### PR DESCRIPTION
**Pull request by mmrugi**
Magnifier feature

### CODE REVIEW

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java [hunk 1]
```diff
@@ -231,8 +237,73 @@ public void actionPerformed(ActionEvent event) {
...
         });
         this.add(selectPrevious);
-
+        
+		magnifier.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent event) {
+				SwingUtilities.invokeLater(new Runnable() {
+					@Override
+					public void run() {
+						if (isMagnifierOpened()) {
```
Comment (RoughTomato) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java]: brackets need to be compatible with the rest of the file.
Both if and else should start within new line.

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java [hunk 2]
```diff
@@ -231,8 +237,73 @@ public void actionPerformed(ActionEvent event) {
...
+							closeMagnifierFrame();
+						} else {
```
Comment (RoughTomato) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/EditMenu.java]: same here

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/MagnifierFrame.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+	public Timer getMouseMoveTimer() {
+		return mouseMoveTimer;
+	}
+
+	/**
+	 * Increase zoom level
+	 */
+	private void increaseZoomLevel() {
+		if (zoomFactor == ZOOM_LEVEL_1) {
+			zoomFactor = ZOOM_LEVEL_2;
```
Comment (RoughTomato) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/MagnifierFrame.java]: You might want to consider changing those similarly to ones in EditMenu.java. @NdV66 @abizukojc what's your opinion?

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/MenuFactory_de.properties [hunk 4]
```diff
@@ -86,6 +86,8 @@ edit.select_next.accelerator=ctrl RIGHT
 edit.select_previous.text=Vorheriges Ausw\u00e4hlen
 edit.select_previous.mnemonic=V
 edit.select_previous.accelerator=ctrl LEFT
+edit.magnifier.text=Vergr��erungsglas
```
Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/MenuFactory_de.properties]: Use Unicodes for national chars

**Review by abizukojc — CHANGES_REQUESTED** [review]

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/MenuFactory_de.properties (hunk 4 above)
Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/MenuFactory_de.properties]: Use unicodes for national chars.

**Review by RoughTomato — APPROVED** [review]

**Review by abizukojc — APPROVED** [review]

**Pull request merged**